### PR TITLE
Changed code samples in readme

### PR DESCRIPTION
**Pull request by ayansengupta17**
1. Removed semi-colons from both ANN and CNN code. (Not required in python)
2. CNN code failed to run as proper import statements from Keras were missing. Added the required code.

### CODE REVIEW

**Comment by ayansengupta17** [comment]
Well yes, I am improving the main code as well. Your CNN Code in readme will fail to run as proper imports were not mentioned eg: you need to import Dense, Dropout, Flatten etc.

**Comment by Prodicode** [comment]
The code is an example and doesn't affect the actual library, therefore we don't consider it a necessary change.

Developers using this library will use it to create their own visualizations for their own neural networks. The code in readme is just an example/PoC.
